tests/Resource/CloudAccount/EntityTest.php

```diff
@@ -77,4 +79,73 @@ public function testSetPhpVersion() {
       'invokes $endpoint->setPhpversion($entity, 7.2)'
     );
   }
+
+  /**
+   * @covers Entity::backup
+   */
+  public function testCreateBackup() {
+    $entity = $this->_getSubject();
+
+    $backup = $this->createMock(Backup::class);
```

Comment:
`$backup` doesn't need to be a mock, does it? Could just be a bare `Backup` instance?

Reply:
Yes, I was learning as I go and I realize that now. :)